mobile/src/main/java/org/openhab/habdroid/core/connection/ConnectionManagerHelper.kt

```diff
@@ -30,18 +30,17 @@ import kotlinx.coroutines.launch
 
 typealias ConnectionChangedCallback = () -> Unit
 interface ConnectionManagerHelper {
-    val currentConnection: ConnectionType
+    val currentConnections: List<ConnectionType>
     var changeCallback: ConnectionChangedCallback?
 
     fun shutdown()
 
-    sealed class ConnectionType {
-        class None : ConnectionType()
-        class Wifi(val network: Network?) : ConnectionType()
-        class Ethernet(val network: Network?) : ConnectionType()
-        class Mobile(val network: Network?) : ConnectionType()
-        class Unknown(val network: Network?) : ConnectionType()
-        class Vpn(val network: Network?) : ConnectionType()
+    sealed class ConnectionType constructor(val network: Network?) {
+        class Wifi(network: Network?) : ConnectionType(network)
```

Comment:
Can you sort them by priority? Just to have a consistent order.

Reply:
I can, but after this patch this class isn't opinionated about priority anymore. The priority of types is handled at the ConnectionFactory level.

Reply:
Ok, than sort them alphabetically?